apps/sms/js/activity_handler.js

```diff
@@ -511,7 +511,11 @@ var ActivityHandler = {
     //
     var params = Utils.params(message.imageURL);
 
+    // When notification is removed from notification tray, notification system
+    // message will still be fired, but "clicked" property will be equal to
+    // false. In this case we should open default ThreadList panel.
     if (!message.clicked) {
+      Navigation.toPanel('thread-list');
```

Comment:
I'm fine with proposal for master, but not sure that we should do it for v2.2 or you think it's safe enough?